src/SFA.DAS.LevyTransferMatching.Infrastructure/Stubs/StubEmployerAccountsApiClient.cs

```diff
@@ -1,75 +1,28 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net.Http;
-using System.Text;
-using System.Threading;
+﻿using System.Threading;
```

Comment:
Do we need the stub now that we use a local CosmosDb, or can it be removed to simplify the codebase?